lib/environment.js

```diff
@@ -19,6 +21,7 @@ module.exports = {
         backbone: {glob: true},
         cheerio: {glob: true},
         assert: {glob: true},
+        'csv-parse': {resolve: 'csv-parse/lib/sync', expose: 'csv-parse', glob: true},
```

Comment:
@shamasis @kunagpal should we expose this as a different name, since we're only providing the synchronous API?